### ISSUE TITLE
Add GitHub integration notes

### ISSUE THREAD

**Issue by netme** [zalando-incubator/zally#54]
Add instructions on how to integrate Zally with Github. 

Currently we should mention a simple TravisCI script located here: https://github.com/zalando-incubator/zally/blob/master/examples/.travis.yml.example 

Later we should add a proper GH integration instructions.

**Comment by fmueller** [comment]
@Nagogus @mkulak Can you add your notes first? Based on this we can write documentation and see what needs to be implemented further.

**Comment by netme** [comment]
Maybe it can be another task then?

**Comment by mkulak** [comment]
Currently GH integration works this way:
* We registered web-hook in [test repo](https://github.com/Nagogus/zally-test) on actions "PR created" and "commit"
* There is special endpoint /github-hook in [GithubController](https://github.com/zalando-incubator/zally/blob/master/server/src/main/java/de/zalando/zally/GithubController.java) which is called by github on each PR/commit in test repo. (see [docs](https://developer.github.com/webhooks/))
* When callback from GH is received we set status of corresponding PR to `pending`. Then download content of swagger file from this PR and validate it. If there are MUST violations we set status of this PR to `failed` (which means that PR cannot be merged). Otherwise we set status to `success`.

Notes on current implementation:
* All requests are executed synchronously (set pending -> get content -> set success/failed)
* User need to setup webhook manually (Zappr does this automatically)
* In order to access file content we need to get token for this repo. For now it's hardcoded (same thing).
* Path to swagger file is hardcoded to `swagger/swagger.yaml` (perhaps there will be option to specify this path in Zally's frontend)
* Status for PR is called "commit status" in GH parlance.
